content/en/docs/reference/using-api/deprecation-guide.md

```diff
@@ -60,6 +69,12 @@ The **events.k8s.io/v1beta1** API version of Event will no longer be served in v
     * use `reportingController` instead of the deprecated `source.component` field (which is renamed to `deprecatedSource.component` and not permitted in new **events.k8s.io/v1** Events)
     * use `reportingInstance` instead of the deprecated `source.host` field (which is renamed to `deprecatedSource.host` and not permitted in new **events.k8s.io/v1** Events)
 
+#### HorizontalPodAutoscaler {#horizontalpodautoscaler-v125}
+
+The **autoscaling/v2beta1** API version of HorizontalPodAutoscaler will no longer be served in v1.25.
+
+* Migrate manifests and API clients to use the **autoscaling/v2** API version, available since v1.23.
```

Comment:
include `* All existing persisted objects are accessible via the new API` and let's merge this to get this update available... there are API changes from v1beta1 → v2 but I'd like @josephburnett to enumerate those in a follow-up

Reply:
Added 
>* All existing persisted objects are accessible via the new API